performance/README.md

```diff
@@ -0,0 +1,114 @@
+
+For the best performance, Howso Platform workloads should be split into two node pools.  One for the core services, NATS and datastores, and one for the trainee pods.  This sets up the core services to run permently without interference from the trainee pods, and allows the trainee pods to be scaled independently of the core services.  The core node pool should have a permenant minimum number of nodes, optionally autoscaling (an extra node or 2) to allow for certain services to [scale](#core-service-horizontal-pod-autoscaler) under heavy load.  The trainee node pool should be autoscaling, potentially to a minimum of 0 nodes and up to the largest capacity required for the heaviest workloads.  Only trainee pods should run on the trainee node pool. 
+
+#### Node Pool Configuration
+
+When configuring autoscaling configuration, it may be useful to refer to the [multi-node setup](../prereqs/README.md#multi-node-k3d-cluster) which demonstrates the use of labels and taints (in a local setup) to control pod scheduling across multiple nodes.
+
+##### Core Services Nodes
+
+The core services of Howso Platform and its dependent charts should run on 2 or 3 dedicated cluster nodes utilizing a [node label](../prereqs/README.md#multi-node-k3d-cluster), to keep trainee pods from starting on them.
+
+```sh
+kubectl label nodes k3d-platformk8s-server-0 howso.com/allowWorkers=False --overwrite
+```
+
+Note, since some core services are set up to [auto-scale](#core-service-horizontal-pod-autoscaler) the core service nodes can be set to to autoscale to accomodate any increased demand under load.  They should not be allowed to scale below the minimum number of nodes required to run the core services.
+
+##### Trainee/Worker Pod Nodes
+
+A node pool of worker nodes should be created; these nodes should be tainted with `howso.com/nodetype=worker:NoSchedule` allowing trainee pods to run on them.
+
+The worker node pool should be configured with cluster autoscaling to automatically add and remove nodes as needed.
+
+The minimum number of nodes can be 0 for maximum efficiency, but it is worth considering a miniumum of 1 worker nodes so that basic test workloads do not have to wait for a new node to be created.
+
+
+#### Create wait time
+
+By default, the Howso Platform client will wait 30 seconds for a trainee to be created, before erroring.  If the cluster is autoscaling (or slow), this is not enough time for a new node to come online and the trainee pod to start.
+
+Increase the create wait time either via environment variable or in the client configuration (howso.yaml).  The value is in seconds, and 0 will wait indefinitely (though will be beholden to server timeouts).
+
+```sh
+export HOWSO_CLIENT_CREATE_MAX_WAIT_TIME=1200
```

Comment:
In a Howso Platform setup, is this environment-variable setting visible anywhere?

Reply:
Added a note to make the usage clearer